frontend/src/metabase/databases/components/DatabaseDetailField/DatabaseDetailField.tsx

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import { EngineField } from "metabase-types/api";
+import FormNumericInput from "metabase/core/components/FormNumericInput";
+import FormFileInput from "metabase/core/components/FormFileInput";
+import FormInput from "metabase/core/components/FormInput";
+import FormSelect from "metabase/core/components/FormSelect";
+import FormTextArea from "metabase/core/components/FormTextArea";
+import FormToggle from "metabase/core/components/FormToggle";
+import DatabaseInfoField from "../DatabaseInfoField";
+import DatabaseSectionField from "../DatabaseSectionField";
+import { FIELD_OVERRIDES } from "../../constants";
+import { EngineFieldOverride } from "../../types";
+
```

Comment:
Should we follow the rule that the closer the import is to the source file, the lower it is in the imports list? 
If we do, then `constants` and `types` should go before `DatabaseInfoField` and `DatabaseSectionField`